### DISCUSSION TITLE
Can I disable error.vue when running in dev mode?

### DISCUSSION THREAD

**Discussion by joneath** [nuxt/nuxt#25158] [Questions]
I have a custom error page at the root of my project `error.vue` and while this works great in production I find I miss the verbose dev error page. I would love a way to disable this `error.vue` page when running in dev mode, is this possible?

My current workaround is to just `console.error(error)` in the `error.vue` page's setup which generally works but a new dev on the team had a project config error that was getting swallowed up by the error page and not even being `console.error` logged...

**Comment by huang-julien** [comment]
Hey :wave:

Do you ahve a small reproduction about the console.error being swallowed (this maybe a bug) ?
Even in dev mode you should be able to get the verbose error. Here's the exemple of the default nuxt error page 
https://github.com/nuxt/nuxt/blob/main/packages/nuxt/src/app/components/nuxt-error-page.vue
You can also import it in dev mode to only rendered it if you wish. 
`const LazyDefaultNuxtError = defineAsyncComponent(() => import('#app/components/nuxt-error-page.vue')`


https://stackblitz.com/edit/nuxt-starter-xymieu?file=pages%2Findex.vue,error.vue,app.vue

**Reply by joneath** [reply]
Perfect, this is exactly what I was hoping for and it works great!

As for a reproduction of the error swallowing, I attempted to create the same conditions as the new team member's setup but I couldn't replicate the issue. It was very strange because the initial server side render and then client side hydration worked without any errors but then on a subsequent hard refresh the server side rendering would fail in a 500 with no error being exposed. 

If it comes up again with any other new team member then I'll make sure to look deeper into it and try and share a reproduction.

Thanks for the help!

**Reply by huang-julien** [reply]
Thank you ! Feel free to open a bug once reproduced !
